tuxemon/core/states/monster/__init__.py

```diff
@@ -118,12 +121,12 @@ def refresh_menu_items(self):
                 monster = self.game.player1.monsters[index]
             except IndexError:
                 monster = None
+                continue
             item.game_object = monster
-            item.enabled = monster is not None
+            item.enabled = monster is not None and self.is_valid_entry(monster)
```

Comment:
Should we be checking the user?  Does the user have any condition that would disable the item?

Reply:
I had initially intended to support that, but there are currently no conditions to check.
It would also be bad UX to have that check happen in the monster menu; it should be handled in the Item menu.

Reply:
fair enough